gazelle_haskell_modules/lang.go

```diff
@@ -215,10 +214,47 @@ func (*gazelleHaskellModulesLang) Fix(c *config.Config, f *rule.File) {
 				r.Delete()
 			}
 		}
+		cleanupModulesList(r, ruleNameSet)
+		cleanupHiddenModulesList(r, ruleNameSet)
 	}
+
 	f.Sync()
 }
 
+func cleanupModulesList(r *rule.Rule, ruleNameSet map[string]bool) {
+	// TODO: use labels instead of manually stripping away the ':' ?
+	stripColon := func(module string) string { return module[1:] }
+	cleanupModulesLists(r, ruleNameSet, "modules", stripColon)
+}
+
+func cleanupHiddenModulesList(r *rule.Rule, ruleNameSet map[string]bool) {
+	ruleName := r.Name()
+	// TODO: use something better?
```

Comment:
Not sure if this isn't already implemented somewhere around here, but I didn't find it.